src/extended-schema/extended-schema.ts

```diff
@@ -6,14 +6,53 @@ export interface FieldMetadata {
 }
 
 export interface LinkConfig {
+    /**
+     * The field name on the target type that will be used to populate the link.
+     * 
+     * @type {string}
+     * @memberof LinkConfig
+     */
     field: string
+    /**
+     * The argument to use when requesting data from the target field.
+     * 
+     * @type {string}
+     * @memberof LinkConfig
+     */
     argument: string
+    /**
+     * When requesting mutiple items from a target field that supports an argument list set batchMode to "true" to submit all argument values in a single batch.
+     * 
+     * @type {boolean}
+     * @memberof LinkConfig
+     */
     batchMode: boolean
+    /**
+     * When using batchMode, this specifies the field on the source type to gather agument values to be used in the batched request
```

Comment:
Cool that you also added the other comments :+1: You got this one wrong, however - *When using batchMode, you can set this to a field in the target type which will be compared against the key values to match target objects to the link locations. If omitted, the elements returned by the target fields are assumed to be in the same order as the keys given as argument value.*